src/main/java/com/google/devtools/build/lib/buildeventservice/BuildEventServiceOptions.java

```diff
@@ -34,6 +34,8 @@
     help =
         "Specifies the build event service (BES) backend endpoint as HOST or HOST:PORT. "
             + "Disabled by default."
+            + "The supported schemas are grpc and grpcs (grpc with TLS enabled). "
+            + "If no schema is provided bazel'll default to grpcs. Specify grpc:// schema to disable TLS."
```

Comment:
This seems to imply that bes_backend should default to grpcs, but from what I've seen it doesn't.
Where's the disconnect?

Reply:
It is, starting from version 1.0 (it is not a part of 0.29, I can see that there was a confusion around this, sorry for that). Before 1.0, in order to get this behaviour one have to specify `--incompatible_tls_enabled_removed` flag.
Starting 1.0 this won't be necessary.
see #8061 for details.

Reply:
Ooh, I missed that! Yes, it works with Bazel 1.0 release candidate!